azure_sdk/resources/resource_group.py

```diff
@@ -23,8 +23,9 @@ class ResourceGroup(AzureResource):
 
     def __init__(self, azure_config, logger,
                  api_version=constants.API_VER_RESOURCES):
-        super(ResourceGroup, self).__init__(azure_config)
         self.logger = logger
+        self.logger.info('Sending these credentials: {}'.format(azure_config))
```

Comment:
is this a good idea , to keep this log of the credentials here , or it was for debugging purpose only